.gitignore

```diff
@@ -1,3 +1,4 @@
+*.DS_Store
```

Comment:
did I create a new gitignore?! lol that was an accident. thanks for the link though! I'll remove it